src/io/flutter/view/FlutterView.java

```diff
@@ -429,35 +438,24 @@ protected void handleJxBrowserInstalled(FlutterApp app, InspectorService inspect
 
   private void presentDevTools(FlutterApp app, InspectorService inspectorService, ToolWindow toolWindow, boolean isEmbedded) {
     assert(SwingUtilities.isEventDispatchThread());
-    final DevToolsManager devToolsManager = DevToolsManager.getInstance(app.getProject());
 
-    if (devToolsManager.hasInstalledDevTools()) {
-      addBrowserInspectorViewContent(app, inspectorService, toolWindow, isEmbedded);
-    }
-    else {
-      presentLabel(toolWindow, INSTALLING_DEVTOOLS_LABEL);
-      awaitDevToolsInstall(app, inspectorService, toolWindow, isEmbedded, devToolsManager);
-    }
+    presentLabel(toolWindow, INSTALLING_DEVTOOLS_LABEL);
+
+    openInspectorWithDevTools(app, inspectorService, toolWindow, isEmbedded);
   }
 
-  protected void awaitDevToolsInstall(FlutterApp app, InspectorService inspectorService, ToolWindow toolWindow, boolean isEmbedded, DevToolsManager devToolsManager) {
+  protected void openInspectorWithDevTools(FlutterApp app, InspectorService inspectorService, ToolWindow toolWindow, boolean isEmbedded) {
     ApplicationManager.getApplication().executeOnPooledThread(() -> {
-      final CompletableFuture<Boolean> result = devToolsManager.installDevTools();
+      final DevToolsInstance instance = DevToolsService.getInstance(myProject).getDevToolsInstance();
```

Comment:
simpler pattern:
rather than starting a pooled thread just call
`AsyncUtils.whenCompleteUiThread(DevToolsService.getInstance(myProject).getDevToolsInstance(), ...)`
and remove the invokeLater.